app/services/trials.js

```diff
@@ -99,10 +100,6 @@ function _processData(trials) {
       return result;
     });
 
-    results = _.sortBy(results, function(item) {
-      return item.publicationDelay;
-    });
-
```

Comment:
With the current code, the sort by publication delay isn't working correctly. The expected behaviour is that the most delayed trials will appear on top (i.e. sorted in descending order for delay). I'm not sure if this code is related to the issue, but I'd start looking there.

The expected behaviour is that, when you open the page, it's sorted by delay in descending order. Also test that changing the sorting order works as expected.